src/db/group.rs

```diff
@@ -32,6 +32,9 @@ pub struct Group {
     /// The list of time fields for this group
     pub times: Times,
 
+    // Custom Data
+    pub custom_data: CustomData,
```

Comment:
@varjolintu I think this would have been a good candidate for a separate PR. Adding the custom data to the group struct looks like a self-contained change that wasn't necessarily related to the other changes in this PR. Moving that to a separate PR would have made reviewing this PR easier.

Reply:
I agree.